### ISSUE TITLE
Please tell us about the future activities of this repository. Or let's discuss it!

### ISSUE THREAD

**Issue by aiji42** [MrBlenny/react-flow-chart#223]
Hello everyone.
I would like to implement this package into my own product. However, there has been no activity in this repository for 6 months, updates have stopped, and I am hesitant to implement it.

I've also contributed to the project, but since it's not clear how to contribute (pull requests are not resolved), I think it might be better to fork it and resume activities, what do you think? Or, if there are already more active repositories derived from this project, please let me know.

I've already compared it with react-flow-renderer, but that one doesn't have the drag-and-drop feature to add nodes, and this project seems to have more features.

**Comment by designviacode** [comment]
@aiji42 I agree with your questions and would like to voice the same concerns.. 
exploring using this package in a large-project, so, kinda need some clarity on this.

**Comment by tahmid-shahriar-lincx** [comment]
@aiji42 thanks for putting out this concern. I have incorporated this package in one of our applications, would love to see more activities in this repository as well.

**Comment by sezanzeb** [comment]
I think it's abandoned, MrBlenny only contributes to some private repos at the moment.

Somebody else probably has to fork it, handle all the open PRs and start maintaining it. But I don't know how one would communicate that the fork exists and that it is actively maintained.

Unless @MrBlenny adds one or more maintainers, possibly out of those who already contributed.

**Comment by aiji42** [comment]
Thanks everyone.

There are 227 forks in this repository. Some of them have a few stars, but no repository has a prominent endorsement yet.
I think that having so many forks means that there are so many supporters, and that there are improvements and ideas.
Shouldn't we integrate it into a new repository as soon as possible and make an announcement to that effect? This project needs someone's lead.

**Comment by MrBlenny** [comment]
Yep, sorry... I've been way too busy with other stuff lately :(

Happy to add someone as a maintainer.

**Comment by davidanitoiu** [comment]
I'd be interested to be the maintainer.

Already had a look at some of the code and I'm using it actively in some of my projects.

**Comment by jwaltgrant** [comment]
@MrBlenny I have been working with this repo. I'd love to join maintaining it. I've done a deep integration with Redux successfully as well as developed a custom context for state management.

**Comment by MrBlenny** [comment]
Hey sorry! I've been very slack. Haven't been finding much time to work on open-source lately :(
I'd be happy to transfer maintenance to either of you.
@jwaltgrant  @davidanitoiu
